### ISSUE TITLE
[UTXO] how can i select utxo for inscribe oridinals on bitcoin?

### ISSUE THREAD

**Issue by weixuefeng** [trustwallet/wallet-core#3420]

**Comment by lamafab** [comment]
Hello @weixuefeng, can you clarify the questions precisely? I have a rough overview here: https://github.com/trustwallet/wallet-core/pull/3233#issuecomment-1601003573

**Comment by weixuefeng** [comment]
eg:
i have utxos
[{txid: "a", amount: 1000, index: 1}, {txid: "a", amount: 2000, index: 1}, {txid: "a", amount: 3000, index: 1}]

then i want to inscribe oridinals, assume i need 1000 sats fee, which utxos i need, and how can i choose them? wallet-core support it now?

**Comment by lamafab** [comment]
Yes,  you need to select the inputs and outputs manually, [and carefully, especially for the tagged satoshis](https://github.com/trustwallet/wallet-core/pull/3233#issuecomment-1601003573). While I'm currently trying to improve this on our `dev` branch, right now you need to determine the necessary fee yourself. You can use the `TWBitcoinFeeCalculateFee` function for that, which takes the full, encoded Bitcoin transaction.

So in other words:

* Create a (dummy) reveal transaction, determine fee.
* Create the reveal transaction that considers the reveal fee.
* Create the final reveal transaction.

**Comment by weixuefeng** [comment]
thanks for reply
